src/app/submit-item/components/submit-page/submit-page.component.html

```diff
@@ -129,7 +115,8 @@ <h3>E-Mail-Adresse</h3>
         <div class="submit-check-container">
           <h2>Rechtliche Angaben</h2>
           <p>
-            Zum Abschluss der Einreichung, hinterlasse uns bitte deine E-Mail-Adresse und bestätigte die Datenschutzerklärung und die
+            Zum Abschluss der Einreichung, hinterlasse uns bitte deine E-Mail-Adresse und bestätigte die
+            Datenschutzerklärung und die
             Nutzungsbedingungen.
```

Comment:
There is no comma in this sentence:
"Zum Abschluss der Einreichung hinterlasse uns bitte deine E-Mail-Adresse ..."
Avoid using "und" twice, 
"bestätigte" -> "bestätige"
To shorten the sentence we could omit the definite article?
"Zum Abschluss der Einreichung hinterlasse uns bitte deine E-Mail-Adresse und bestätige Datenschutzerklärung sowie Nutzungsbedingungen."